src/analysis/child_properties.rs

```diff
@@ -44,11 +44,7 @@ pub fn analyze(
         return properties;
     }
     let config = config.unwrap();
-    let child_name = config
-        .child_name
-        .as_ref()
-        .map(|s| &s[..])
-        .unwrap_or("child");
+    let child_name = config.child_name.as_ref().map_or("child", |s| &s[..]);
```

Comment:
`.as_str()` please instead of `&s[..]`.

Reply:
Fixed

Reply:
There are a few more of these in the code-base - would you mind fixing those up as well?

Reply:
Fixed. In two cases clippy then suggested to make it even simpler :)